docs/docs/using-gatsby-image.md

```diff
@@ -40,7 +40,7 @@ With Gatsby, we can make images way way better.
 1. Install `gatsby-image` and other, necessary dependencies like `gatsby-plugin-sharp` and `gatsby-transformer-sharp`
 
 ```shell
-  npm i gatsby-image gatsby-transformer-sharp gatsby-plugin-sharp
+  npm i --save gatsby-image gatsby-transformer-sharp gatsby-plugin-sharp
```

Comment:
```suggestion
  npm install --save gatsby-image gatsby-transformer-sharp gatsby-plugin-sharp
```

Sure - it's more explicit, which I suppose is better. If we can make this change we can get this merged!